.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
 
 before_install:
   - sudo sh setup.sh
+  - npm config set registry http://ci.strongloop.com:4873/
```

Comment:
This registry isn't meant to be accessed externally. If this repo depends on unpublished upstream changes, then those changes _need_ to be published.